cdk/lib/dashboard.ts

```diff
@@ -86,26 +94,26 @@ export class CacclMonitoring extends Construct {
 
     const httpCodeWidgets = ['2', '3', '4', '5'].map((i) => {
       const metricName = `HTTP ${i}xx Count`;
-      const httpCode = `TARGET_${i}XX_COUNT` as keyof typeof HttpCodeTarget;
-      return new GraphWidget({
+      const httpCode = `TARGET_${i}XX_COUNT` as keyof typeof elb.HttpCodeTarget;
```

Comment:
Wow this is some fancy typescript! I wonder if there's a simpler way, but maybe not.